### ISSUE TITLE
Anybody trained successfully without pretrain?

### ISSUE THREAD

**Issue by pyjennings** [aloyschen/tensorflow-yolo3#7]
How about the loss? Thanks!

**Comment by sanmianjiao** [comment]
you mean that you train this model without darknet53 weights?

**Comment by pyjennings** [comment]
@sanmianjiao yes, can I?

**Comment by sanmianjiao** [comment]
I did it, but failed  :(
